modules/cluster-logging-release-notes-5.3.z.adoc

```diff
@@ -1,5 +1,48 @@
 //Z-stream Release Notes by Version
 
+[id="cluster-logging-release-notes-5-3-4"]
+= OpenShift Logging 5.3.4
+This release includes link:https://access.redhat.com/errata/RHBA-2022:0411[RHBA-2022:0411 OpenShift Logging Bug Fix Release 5.3.4]
+
+[id="openshift-logging-5-3-4-bug-fixes"]
+== Bug fixes
+* Before this update, changes to the metrics dashboards had not yet been deployed because the cluster-logging-operator did not correctly compare existing and desired configmaps containing the dashboard.  This update fixes the logic by adding unique hash value to the object labels. (link:https://issues.redhat.com/browse/LOG-2066[LOG-2066])
+
+* Before this update, Elasticsearch pods failed to start after updating with FIPS enabled. With this update, Elasticsearch pods start successfully. (link:https://issues.redhat.com/browse/LOG-1974[LOG-1974])
+
+* Before this update, elasticsearch generated the error "Unable to create PersistentVolumeClaim due to forbidden: exceeded quota: infra-storage-quota." if the PVC already existed. With this update, elasticsearch checks for existing PVCs, resolving the issue. (link:https://issues.redhat.com/browse/LOG-2127[LOG-2127])
```

Comment:
Same two comments (parentheses and lower-case 'elasticsearch') for modules/cluster-logging-release-notes-5.2.z.adoc also apply to this line

Reply:
s/elasticsearch/Elasticsearch/?

Reply:
Migrate updates from previous release notes to similar ones, like this one, further down.